bats/fb-content-katello.bats

```diff
@@ -121,9 +121,10 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
-  rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm || true
-  subscription-manager register --force --org="Test_Organization" --activationkey="Test AK" || true
-  subscription-manager status | grep -q "Current"
+  run yum erase -y 'katello-ca-consumer-*'
+  run rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm
+  run subscription-manager register --force --org="Test_Organization" --activationkey="Test AK"
```

Comment:
`run` ignores the exit code (basically it's the same as `|| true` here, let's ignore the fact that it sets vars for further analyzing the exit code and output) so it won't fail = no change to old behavior.

I agree that adding those as further tests is a good idea, but
* they were not used as such before this PR
* I am not sure what `sub-man` exits if it cannot add *all* installed product subs

Given that we are checking that we end up with a sub for "Test Product", I think it is safe to assume that the steps before were OK and not try to check their exit codes (at least at the moment).